src/main/java/org/kiwiproject/registry/eureka/client/EurekaParser.java

```diff
@@ -15,10 +15,10 @@
 import org.kiwiproject.net.KiwiInternetAddresses;
 import org.kiwiproject.registry.eureka.common.EurekaInstance;
 
-import javax.annotation.Nullable;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
+import javax.annotation.Nullable;
```

Comment:
Huh....why did this import move?

Reply:
I'm guessing because my settings are probably a little off on this machine